preprocessing/src/main/java/ImageData.java

```diff
@@ -25,8 +25,8 @@
     transient private MatOfByte compressed_img;
 
     //Status of the representation
-    public enum ImageDataState {UNCOMPRESSED, COMPRESSED, ERROR}
-    private ImageDataState state;
+    //public enum ImageDataState {UNCOMPRESSED, COMPRESSED, ERROR}
+    private main.java.MetaImageData.ImageDataState state;
```

Comment:
This element exists already in the class above, does not make sense to redefine it here